### PR TITLE
cleanup(scheduler): move metric labels to metrics package

### PR DESCRIPTION
**Pull request by sanposhiho**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

move metric labels to `metrics` package.

#### Which issue(s) this PR fixes:

follow up: https://github.com/kubernetes/kubernetes/pull/115082#discussion_r1127028031

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Review by alculquicondor — COMMENTED** [review]
/approve

File: pkg/scheduler/metrics/metrics.go [hunk 1]
```diff
@@ -34,6 +34,21 @@ const (
 	PrioritizingExtender = "prioritizing_extender"
 	// Binding - binding work label value
 	Binding = "binding"
+
```
[the remark this answers is not in the document]
Comment (sanposhiho) [reply pkg/scheduler/metrics/metrics.go:37]: done.

**Comment by sanposhiho** [comment]
Both places aren't using `preFilter` and `Filter` though 🤔 

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/116312/pull-kubernetes-e2e-kind/1632866461603074048
> # k8s.io/kubernetes/pkg/scheduler/framework/runtime
pkg/scheduler/framework/runtime/framework.go:605:60: undefined: preFilter
pkg/scheduler/framework/runtime/framework.go:723:60: undefined: Filter

**Comment by sanposhiho** [comment]
Hmm I have no idea why it fails.
/pkg/scheduler/framework/runtime/framework.go#L605 on the 224bd566a5c1cec1366f7f4f16dc68d84dd2558d isn't using `preFilter`.

https://github.com/kubernetes/kubernetes/blob/224bd566a5c1cec1366f7f4f16dc68d84dd2558d/pkg/scheduler/framework/runtime/framework.go#L605

**Comment by kerthcet** [comment]
The error seems unlikely, let me try again :)
/retest

**Comment by kerthcet** [comment]
Maybe related to the test-infra. 🤔

File: pkg/scheduler/metrics/metrics.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (sanposhiho) [reply pkg/scheduler/metrics/metrics.go:40]: 👍

**Comment by sanposhiho** [comment]
Just pushed another commit, may fix this weird failing.

**Comment by alculquicondor** [comment]
There seem to be compilation errors:

```
W0309 05:34:27.612] pkg/scheduler/framework/runtime/framework.go:605:60: undefined: preFilter
W0309 05:34:27.612] pkg/scheduler/framework/runtime/framework.go:723:60: undefined: Filter
```

**Comment by sanposhiho** [comment]
the weirdness here is no place (including framework.go#L605) isn't using `preFilter` actually.

https://github.com/kubernetes/kubernetes/blob/417cf8f4639a3e7660d985d62cc6d7999a5da2e5/pkg/scheduler/framework/runtime/framework.go#L605

**Comment by alculquicondor** [comment]
rebase?

**Comment by alculquicondor** [comment]
Your commit: https://github.com/kubernetes/kubernetes/blob/42658799c093a2038db80579e50e14fdcf716f5a/pkg/scheduler/framework/runtime/framework.go#L605

**Comment by sanposhiho** [comment]
I see, so those jobs are running after rebasing on the latest master....

**Comment by alculquicondor** [comment]
> I see, so those jobs are running after rebasing on the latest master....

Yes, otherwise we couldn't be sure that this PR doesn't break top-of-tree.
